src/pages/Votes.tsx

```diff
@@ -21,25 +21,120 @@ const votes = [
 	{
```

Comment:
As we talked about yesterday, the data needs to be put in state. But we'll do this next week.